bpo-36227: ElementTree.tostring() default_namespace argument (#12225)
python/cpython: Misc/NEWS.d/next/Library/2019-03-07-20-02-18.bpo-36227.i2Z1XR.rst

> Added new keyword arguments `default_namespace` and `xml_declaration` to functions

Reviewer: ```suggestion
Added support for keyword arguments `default_namespace` and `xml_declaration` in functions
```
